ad/data_source_ad_group.go

```diff
@@ -16,7 +16,7 @@ func dataSourceADGroup() *schema.Resource {
 			"guid": {
 				Type:        schema.TypeString,
 				Required:    true,
-				Description: "The GUID of the Group object.",
+				Description: "The GUID of the Group object. Alternatively it can be the SID, the Distinguished Name, or the SAM Account Name of the group.",
 			},
 			"sam_account_name": {
```

Comment:
Is this needed still if SAM account name works in GUID?

Reply:
This is a computed field. It's used to store the information we get from the host. It's handy if you look for the user by any other attribute, but need the SAM back.